Plugins/Public/mobiledocking_plugin/Main.cpp

```diff
@@ -306,6 +380,22 @@ void __stdcall PlayerLaunch_AFTER(unsigned int ship, unsigned int client)
 		mobiledockClients.erase(client);
 	}
 
+	// Land ship to proxy base in carrier's system if they are in different systems.
+	if (mobiledockClients[client].carrierSystem == -1)
+	{
+		string scProxyBase = HkGetPlayerSystemS(HkGetClientIdFromCharname(mobiledockClients[client].wscDockedWithCharname)) + "_proxy_base";
+		uint iBaseID;
+		if (pub::GetBaseID(iBaseID, scProxyBase.c_str()) == -4)
+		{
+			PrintUserCmdText(client, L"No proxy base in system detected. Contact a developer about this please.");

```

Comment:
add a thing about telling the dev which system they tried to undock in

Reply:
Now it tells to player which base is required.